models/track_and_reid_model.py

```diff
@@ -411,8 +444,8 @@ def create_data_by_re_id_and_track():
 
     all_tracks_final_scores = dict()
 
-    faceClassifer = FaceClassifer(num_classes=21, label_encoder=le, device='cuda:0')
-    faceClassifer.model_ft.load_state_dict(torch.load("/mnt/raid1/home/bar_cohen/FaceData/checkpoints/FULL_DATA_augs:True_lr:1e-05_0, 4.pth"))
+    faceClassifer = FaceClassifer(num_classes=19, label_encoder=le, device='cuda:0')
+    faceClassifer.model_ft.load_state_dict(torch.load("/mnt/raid1/home/bar_cohen/FaceData/checkpoints/4.8 Val, 1.pth"))
     faceClassifer.model_ft.eval()
```

Comment:
maybe instead of replacing these lines, duplicate them and comment out the old face classifier so it will be easier to switch between them if necessary